### DISCUSSION TITLE
Anyway to install a specific version of AdGuard Home uisng your LXC script?

### DISCUSSION THREAD

**Discussion by up2you1** [tteck/Proxmox#1592] [Help Needed]
There seems to be an issue with latest verison of Adguard Home (which the LXC container has updated too). https://github.com/AdguardTeam/AdGuardHome/issues/5896

Id like to downgrade to a previous version of Adguard Home

is this possible?

in docker i can select the tag of adguard/adguardhome:edge or :beta, is there anything like this for LXC?

Thanks!

**Comment by up2you1** [comment]
anyone? the downgrade question is still valid. thanks

**Comment by gigi2006** [comment]
good question.
i have realy big problems whit the actual version and i have no backup.
so i install the pihole now
